src/components/Product.js

```diff
@@ -0,0 +1,14 @@
+import React, {Component} from 'react';
+
+export default class Product extends Component {
```

Comment:
En la mayoria de tus archivos me parece que estas usando funcional components y no class components, me parece que lo mejor sería llevar todos de la misma manera para que el proyecto tengo consistencia

Reply:
The target is using class components for all of them.
I've seen the part of the lifecycle and some of them would implement it without problems. Some others won't, but as you say, for the project consistency all of them will be class components.

Reply:
Good. Just take into consideration that what React is focusing I believe since version 17 moving towards functional components, as components this way are smaller and some will say easier to re-use. From my experience class components trend to be bigger and somewhat a bit more difficult to maintain.

Reply:
Changed to functional components according to Diana's session. I'll take this as a standard.